### PR TITLE
fix(TableHeaderLabel): adds ability to declare a column as unsortable…

### PR DESCRIPTION
**Pull request by pmsorhaindo**
… in order to remove cloaked sorting icon

https://5a5f74c0a6188f16f35500b2--librarian-chimpanzee-34483.netlify.com/docs/components/table?state=%7B%22TableHeaderLabel%22%3A%7B%22props%22%3A%7B%22sortable%22%3Afalse%7D%7D%7D

### CODE REVIEW

File: src/components/table/TableHeaderLabel.test.js [hunk 1]
```diff
@@ -36,4 +36,10 @@ describe('TableHeaderLabel', () => {
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot();
   });
+
+  it('renders with sortable', () => {
+    const component = getComponent({ unsortable: false });
```
Comment (HHogg) [inline src/components/table/TableHeaderLabel.test.js]: Should this say `sortable`?

**Comment by pmsorhaindo** [comment]
@HHogg I went with sortable to avoid `!unsortable && ...`. as I think it reads nicer. If you disagree with sortable being `true` by default I'm happy to change it.

File: src/components/table/TableHeaderLabel.js [hunk 2]
```diff
@@ -16,12 +16,16 @@ export default class TableHeaderLabel extends Component {
...
+    /** Indicates the column is sortable if false it will remove the cloaked icon
+    indicating sort direction */
+    sortable: PropTypes.bool,
     /** Set text-align */
     textAlign: PropTypes.oneOf(['left', 'right']),
   };
 
   static defaultProps = {
     textAlign: 'left',
+    sortable: true,
```
Comment (HHogg) [inline src/components/table/TableHeaderLabel.js]: Even though the previous behaviour defaulted to adding the Icon whether sorting was on or not (which was incorrect), do you think it makes more sense if columns weren't sortable by default?

Comment (pmsorhaindo) [reply src/components/table/TableHeaderLabel.js]: I was thinking that most tabular data should be sortable, a column of checkboxes is rarer than a column of prices, names or dates. Happy to change it though.

**Pull request merged**